packages/react-components/react-charts-preview/src/components/DonutChart/DonutChart.tsx

```diff
@@ -11,6 +11,8 @@ import { ScaleOrdinal } from 'd3-scale';
 import { useId } from '@fluentui/react-utilities';
 import { useFocusableGroup } from '../../../../react-components/src/index';
 import PopoverComponent from '../CommonComponents/Popover';
+import { ThemeContext_unstable as V9ThemeContext } from '@fluentui/react-shared-contexts';
```

Comment:
Are you able to use tokens.colorNeutralForeground1 and tokens.colorNeutralBackground1 directly without needing the theme context

Reply:
That is not working

Reply:
I think to get the component current theme, we will have to use theme context.

Reply:
what is the error. Try using var function like in this file. https://github.com/microsoft/fluentui/blob/854fe51c4fb889cfd0a5e3419cf88ee8c2e837d7/packages/web-components/src/theme/design-tokens.ts#L7

Reply:
The above method using 'var', it will not fetch the exact value hold by this variable. That is the problem. So tried few other ways to access css variables from javascript. That also did not work. So reverted this change for now.